### PR TITLE
1323 iot tracking links

### LINKED ISSUE #1323: IoT page link tagging update
update the following link with a new tagging convention

1: Digital signage on Ubuntu  › - link update to: www.ubuntu.com/internet-of-things/digital-signage?from=iotpage
2: Robots and drones, rocking Ubuntu Core › - link update to: www.ubuntu.com/internet-of-things/robotics?from=iotpage
3: IoT gateways on Ubuntu › - link update to: www.ubuntu.com/internet-of-things/gateways?from=iotpage
4: Learn how to make a snap - link update to: snapcraft.io?from=iotpage
5: Read the AeroLion case study - link update to: pages.ubuntu.com/AerolionCS.html?from=iotpage
6: Read the Azeti Networks case study - link update to: pages.ubuntu.com/AzetiCS.html?from=iotpage
7: Learn about developing with Ubuntu Core - link update to: developer.ubuntu.com/core?from=iotpage
8: Get in Touch button - link update to: www.ubuntu.com/internet-of-things/contact-us?from=iotpage

---
*Reported from: https://www.ubuntu.com/internet-of-things*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* removed utm tracking from all links, except brightalk
* added ?from=iotpage tracking for external links

## QA

1. go to /internet-of-things and confirm links with Done and [copy doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#)

## Issue / Card

Fixes #1323

### CODE REVIEW

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -98,7 +98,7 @@
                 <li>Easy to create and distribute</li>
                 <li>Widely adopted by industry leaders</li>
             </ul>
-            <p><a href="http://snapcraft.io/" class="external">Learn how to make a snap</a></p>
+            <p><a href="http://snapcraft.io?from=iotpage" class="external">Learn how to make a snap</a></p>
```
Comment (WillMoggridge) [inline templates/internet-of-things/index.html]: This link should be `http://snapcraft.io/?from=iotpage` with the forward slash in the middle.

Comment (pmahnke) [inline templates/internet-of-things/index.html]: @WillMoggridge for the index page too?

Comment (pmahnke) [inline templates/internet-of-things/index.html]: and they both work

**Comment by WillMoggridge** [comment]
:D <3

**Pull request merged**